handbook/communication/content_guidelines/actionable_language.md

```diff
@@ -0,0 +1,556 @@
+
+Give users confidence about what they should do.
+
+<div class="usage">
+<div class="item yes">
+<h5>Yes</h5>
+<ul>
+<li>Add your repos from your code host and start searching code.</li>
+</ul>
+</div>
+<div class="item no">
+<h5>No</h5>
+<ul>
+<li>Add your repos from your code host and you can start searching code.</li>
+</ul>
+</div>
+</div>
+
+## Buttons
+
+Buttons are one of the most important elements in our digital platforms. Most actions and conversions are the direct result of the user clicking a button, and the labels used in buttons help users understand what actions are possible and are at the point when a decision becomes an action.
+
+Always write button labels in sentence case.
+
+<div class="usage">
+<div class="item yes">
+<h5>Yes</h5>
+<ul>
+<li>Create new campaign</li>
+</ul>
+</div>
+<div class="item no">
+<h5>No</h5>
+<ul>
+<li>Create New Campaign</li>
+</ul>
+</div>
+</div>
+
+### Buttons for action
+
+When buttons enable user actions, the formula is `verb` + `noun`.
+
+<div class="usage">
+<div class="item yes">
+<h5>Yes</h5>
+<ul>
+<li>Add repo</li>
+<li>Explore extensions</li>
+</ul>
+</div>
+<div class="item no">
+<h5>No</h5>
+<ul>
+<li>Add</li>
+<li>Explore</li>
+</ul>
+</div>
+</div>
+
+### Buttons in confirmations
+
+When buttons are used as calls to action in confirmations, the formula `verb` + `noun` doesn’t have to be followed. Instead, use a verb to clearly convey outcomes, by connecting action to outcome.
+
+<div class="usage">
+<div class="item yes">
+<h5>Yes</h5>
+<ul>
+<li>Discard changes? <strong>Cancel / Discard</strong></li>
+</ul>
+</div>
+<div class="item no">
+<h5>No</h5>
+<ul>
+<li>Discard changes? <strong>No / Yes</strong></li>
+</ul>
+</div>
+</div>
+
+### Buttons for conversion
+
+When buttons support conversion goals, the formula is `value` + `relevance`.
+
+Instead of writing about what the user does (the action), write about what the user gets (the value). Then, make it more immediately relevant to the specific conversion context.
+
+The length of a button label when the button’s purpose is conversion isn’t usually a problem.
+
+<div class="usage">
+<div class="item yes">
+<h5>Yes</h5>
+<ul>
+<li>Start searching code now</li>
+</ul>
+</div>
+<div class="item no">
+<h5>No</h5>
+<ul>
+<li>Try now</li>
+</ul>
+</div>
+</div>
+
+**Further reading:**
+
+- [How to write a call to action that converts](https://unbounce.com/conversion-rate-optimization/how-to-write-a-call-to-action-that-converts-with-case-sudies)
+
+### Buttons as microcopy
+
+Buttons can also be an opportunity to use microcopy to create a more positive experience (particularly in negative situations) by making the interface more human. If it’s appropriate, break the rules and explore ways to match the user’s feelings without assigning emotion.
+
+<div class="usage">
+<div class="item yes">
+<h5>Yes</h5>
+<ul>
+<li>I tried, I’m stuck, I need help</li>
+</ul>
+
+Instead of
+
+<ul>
+<li>Get help</li>
+</ul>
+</div>
+</div>
```

Comment:
Any reason we don't use "Yes/No" style here? Seems legitimate and more consistent with others.

Reply:
That's a great question that I might need to clarify in the context...

Basically, it's "Instead of" rather than "Yes/No" because "Get help" is totally viable as a button label. If you were to use it, you wouldn't be doing anything wrong. But you **could** change it to "I tried, I’m stuck, I need help" if you wanted to take it a bit further and use it as microcopy.

Reply:
Hmm, we could still use the words "Instead of" on the right:

<img width="882" alt="Screen Shot 2020-09-23 at 9 06 41 PM" src="https://user-images.githubusercontent.com/2946214/94016487-c3574700-fde0-11ea-9041-d081acfd8897.png">